app/components/EmotionResults/EmotionResults.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import analyzeResponse from '../../lib/analyzeResponse.js';
 
-export const EmotionResults = ({ results, url }) => {
+export const EmotionResults = ({ results, url, pickedEmotion }) => {
   const noResults = () => {
     return (
       <h2 className='no-results'>Take a picture to see your expression!</h2>
     )
   }
 
+  const ifNotNeutralContemptRenderLink = (emotion) => {
+    if (emotion !== 'neutral' && emotion !== 'contempt') {
+      return (
+        <Link className='learn-more-link' to={`/learn/${emotion}`}>Learn more about {emotion}</Link>
+      )
+    }
+  }
+
   const showResults = () => {
     let emotionsForDisplay = analyzeResponse(results);
```

Comment:
The results change, so the output of analyzeResponse changes ... but does that matter? I guess each time it's a new instance of emotionsForDisplay. I'm not mutating it once it's set.